tsconfig.json

```diff
@@ -19,13 +15,9 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "noEmit": true,
+    "noEmit": false,
     "jsx": "react"
   },
-  "include": [
-    "src"
-  ],
-  "exclude": [
-    "node_modules"
-  ]
-}
\ No newline at end of file
+  "include": ["src", "*/*.config.js"],
+  "exclude": ["node_modules"]
+}
```

Comment:
Do you have an **eslint error** in `tsconfig.json` too? 
```js
JSON schema for the TypeScript compiler's configuration file

Cannot write file '/Users/non_const/bindfly.js/src/reportWebVitals.js' because it would overwrite input file.ts
```